googleanalytics/googleanalytics-connector/googleanalytics-connector-1.0.0/org.wso2.carbon.connector/src/main/resources/googleanalytics-goals/getGoals.xml

```diff
@@ -0,0 +1,59 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+   Copyright (c) 2005-2015, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+
+   WSO2 Inc. licenses this file to you under the Apache License,
+   Version 2.0 (the "License"); you may not use this file except
+   in compliance with the License.
+   You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing,
+   software distributed under the License is distributed on an
+   "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+   KIND, either express or implied. See the License for the
+   specific language governing permissions and limitations
+   under the License.
+-->
+<!-- Method Documentation URL: https://developers.google.com/analytics/devguides/config/mgmt/v3/mgmtReference/management/accounts/list -->
+<template name="getGoals" xmlns="http://ws.apache.org/ns/synapse">
+
+    <parameter name="accountId" description="Account ID to retrieve the views." />
+    <parameter name="webPropertyId" description="Returns response with indentations and line breaks." />
+    <parameter name="profileId" description="Alternative to userIp." />
+    <parameter name="goalId" description="The starting index of the resources." />
+    <parameter name="fields" description="option results in response." />
```

Comment:
Follow same standard for all parameter descriptions. Better start with capital letter.